addon/components/frost-select-dropdown.js

```diff
@@ -397,6 +399,7 @@ export default Component.extend({
   },
   /* eslint-enable complexity */
 
+  /* eslint-disable complexity */
```

Comment:
We should not be employing `eslint-disable complexity` in new code.  It should only be used to temporarily address existing code to work around additions or refactors but in new code it should just be written correctly.